vllm/worker/vineyard_llm_cache.py

```diff
@@ -383,9 +384,11 @@ def update_seq_kv_caches(
         start_unload = torch.cuda.Event(enable_timing=True)
         end_unload = torch.cuda.Event(enable_timing=True)
         start_unload.record()
+        # using a cuda staging buffer to avoid the inefficiency of non-contiguous HBM->DRAM memcpy
```

Comment:
Could you please document the use of the cuda staging buffer in PR description?

Reply:
updated